src/mtev_main.c

```diff
@@ -603,19 +603,19 @@ mtev_main(const char *appname,
       pid_t owner;
       if((lockfd = mtev_lockfile_acquire_owner(lockfile, &owner)) < 0) {
         if(!wait_for_lock) {
-          mtevEL(mtev_stderr, MLKV{ MLKV_NUM("errno", errno), MLKV_END },
+          mtevEL(mtev_stderr, MLKV( MLKV_INT64("errno", errno), MLKV_END ),
                  "Failed to acquire lock: %s\n", lockfile);
           if(owner != -1) {
             pid_t groupid = getpgid(owner);
-            mtevEL(mtev_stderr, MLKV{ MLKV_NUM("pid", owner), MLKV_NUM("pgid", groupid), MLKV_END },
+            mtevEL(mtev_stderr, MLKV( MLKV_INT64("pid", owner), MLKV_INT64("pgid", groupid), MLKV_END ),
```

Comment:
I'm pretty sure these values are 32-bits, which is fine, but `pgid` I'm relatively sure is unsinged